apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,33 @@
+
+#include "TutorialServo.h"
+
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs,
+                             float maxPulsewidthInMs)
+    : m_servoPwmOut(servoPin),
+      m_servoRangeInDegrees(servoRangeInDegrees),
+      m_minPulsewidthInMs(minPulsewidthInMs),
+      m_maxPulsewidthInMs(maxPulsewidthInMs) {}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+  if (degrees > m_servoRangeInDegrees) {
+    m_servoPwmOut.pulsewidth_ms(m_maxPulsewidthInMs);
+  } else if (degrees < 0) {
+    m_servoPwmOut.pulsewidth_ms(m_minPulsewidthInMs);
+  } else {
+    m_servoPwmOut.pulsewidth_ms(1 + degrees / m_servoRangeInDegrees);
```

Comment:
You should be using m_minPulsewidthInMs and m_maxPulsewidthInMs in your mapping from degrees to pulsewidth